recipe/meta.yaml

```diff
@@ -14,26 +14,36 @@ source:
      - libca.patch
 
 build:
-  number: 1
-  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  number: 2
+  script: |
+      export NOLIBCA=1
+      echo "Using LIBCA from $PYEPICS_LIBCA"
+      rm -rf epics/clibs
+      "{{ PYTHON }} -m pip install . --no-deps -vv"
+
 
 requirements:
   host:
-      - python
-      - pip
+    - python
+    - pip
   run:
-      - python
-      - pip
-      - numpy
-      - pyparsing
+    - epics-base
+    - numpy
+    - pip
+    - pyparsing
+    - python
 
 test:
   imports:
     - epics
     - epics.autosave
+    - epics.ca
     - epics.compat
     - epics.devices
 
+  commands:
```

Comment:
@hhslepicka may want to pull this line to your PR?

Reply:
I will add it there! Thanks for he heads up